docs/governance-and-automation/automated-workflows/workspace-type-and-reviewers.md

```diff
@@ -0,0 +1,47 @@
+---
+
+description: This article explains which workspace types can have which policies applied and who can be selected as a reviewer.
+
+---
+
+# Policies: Workspace Types and Reviewers
+
+Before applying the policies you've created or enabled in Syskit Point, it's useful to know which workspaces policies can be applied to and who can be selected as reviewers. 
+
+Below you can find details on:
+
+* **Which types of workspaces** (Microsoft Teams, Microsoft 365 Groups, SharePoint Sites, etc.) are supported by each policy
+
+* **Which users can be assigned as reviewers** to complete tasks created by the policy
```

Comment:
dorada da bude možda malo jasnije
* **Which users can be defined as reviewers** to complete tasks when the delegation option is enabled